doc/example-requests/covid19-api/pipeline.http

```diff
@@ -0,0 +1,30 @@
+### PIPELINE EXAMPLE CONFIGURATIONS FOR COVID19API
+
+@baseURL = http://localhost:9000/api/core
+
+## paste datasource id from datasource config response here
+@dataSourceId = "1"
+
+### Store data without modification and trigger a firebase notification
+POST {{baseURL}}/pipelines  HTTP/1.1
+Content-Type: application/json
+
+{
+  "datasourceId": {{dataSourceId}},
+  "transformation":  {
+    "func": "return data;"
```

Comment:
Do we need any transformations to improve data quality?

Reply:
Yes, that would be nice to show possible applications for the transformation but it is a little complicated to implement because, depending on the related datasource id, the data looks very different. I could merge the pipeline and adapter example request files for the covid example and group them together, so it is clear which pipeline relates to which datasource.

Reply:
I created three separate files now to show three example applications of the transformation functions